tools/topology/platform/common/ssp.m4

```diff
@@ -30,6 +30,11 @@ $6
 dnl SSP_QUIRK_LBM 64 = (1 << 6)
 define(`SSP_QUIRK_LBM', 64)
 
+dnl SSP_CC_MCLK_ES 64 = (1 << 6)
+define(`SSP_CC_MCLK_ES', 64)
+dnl SSP_CC_BCLK_ES 128 = (1 << 7)
+define(`SSP_CC_BCLK_ES', 128)
+
 dnl SSP_CONFIG_DATA(type, idx, valid bits, mclk_id, quirks, bclk_delay,
 dnl clks_control, pulse_width, padding)
 dnl mclk_id, quirks, bclk_delay clks_control, pulse_width and padding are optional
```

Comment:
it's incomplete...we need to modify the SSP_CONFIG_DATA macro here

Reply:
not following the feedback @brentlu, what is missing here?
@bardliao I took this code unmodified, not sure what is wrong?

Reply:
@plbossart I think the issue is that https://github.com/thesofproject/sof/commit/636cbefc62c962a80e66306bc6e50850915bd6d5 is already on the main branch, but @brentlu was testing with glk-012-stable-branch branch where https://github.com/thesofproject/sof/commit/636cbefc62c962a80e66306bc6e50850915bd6d5 is not there.